protocols/kad/src/behaviour.rs

```diff
@@ -1273,6 +1310,7 @@ where
                         let addresses = Addresses::new(a);
                         match entry.insert(addresses.clone(), new_status) {
                             kbucket::InsertResult::Inserted => {
+                                self.handle_bootstrap();
```

Comment:
I think here we should do:

```suggestion
                                if !self.any_bootstrap_successful {
                                	self.handle_bootstrap();
                                }
```

Reply:
done